SeaPublicWebsite/Views/StaticPages/PrivacyPolicy.cshtml

```diff
@@ -0,0 +1,184 @@
+@{
+    ViewBag.Title = $"Privacy Notice on {Constants.SERVICE_NAME}";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">
+            Privacy Notice
+        </h1>
+        <p class="govuk-body">
+            This notice sets out how we will use your personal data, and your rights. It is made under Articles 13
+            and/or 14 of the UK General Data Protection Regulation (UK GDPR).
+        </p>
+        <h2 class="govuk-heading-m">
+            YOUR DATA
+        </h2>
+        <p class="govuk-body">
+            This service does not automatically collect or store any personal data about you.
+        </p>
+        <p class="govuk-body">
+            We may process the following personal data:
+        </p>
+        <p class="govuk-body">
+            Your name, email address and any other information sent directly to us via our email feedback function.
+        </p>
+        <p class="govuk-body">
+            We may also collect and process anonymised information as outlined in our
+            <a class="govuk-link" href="@Url.Action(nameof(CookieController.CookieSettings_Get), "Cookie")">
+                cookie policy
+            </a>.
+        </p>
+
+        <h2 class="govuk-heading-s">
+            Purpose
+        </h2>
+        <p class="govuk-body">
+            We are processing your personal data in order to receive feedback about this service and to contact 
+            you for clarifications or to report service updates.
+        </p>
+        
+        <h2 class="govuk-heading-s">
+            Legal basis of processing
+        </h2>
+        <p class="govuk-body">
+            The legal basis for processing your personal data is processing necessary for the performance of a task
+            carried out in the public interest or in the exercise of official authority vested in the data controller,
+            such as the exercise of a function of the Crown, a Minister of the Crown, or a government department; the
+            exercise of a function conferred on a person by an enactment; the exercise of a function of either House of
+            Parliament; or the administration of justice. In this case, in order to receive feedback about this service
+            and to contact you for clarifications or to report service updates.
+        </p>
+        
+        <h2 class="govuk-heading-s">
+            Recipients
+        </h2>
+        <p class="govuk-body">
+            Your personal data will be shared by us with other Government Departments or public authorities where
+            necessary. We may share your data if we are required to do so by law, forexample by court order or to
+            prevent fraud or other crime.
+        </p>
+        <p class="govuk-body">
+            As your personal data will be stored on our IT infrastructure it will also be shared with our data
+            processors Microsoft and Amazon Web Services.
```

Comment:
We don't use Amazon Web Services anymore
What data does Microsoft get? If you refer to Google analytics, that is google not microsoft and I don't think it's shared with them (I'm not an expert on this however)

Reply:
This is about emails sent to BEIS by the feedback function. I assume that they are stored by Microsoft in Exchange365. I have queried AWS being mentioned here, but I'd like to get this deployed so that BEIS can review it in a form that makes sense to them. I expect there to be a follow-up ticket for the exact wording.